sway/ipc-json.c

```diff
@@ -421,6 +421,7 @@ static void ipc_json_describe_container(struct sway_container *c, json_object *o
 		view_is_urgent(c->view) : container_has_urgent_child(c);
 	json_object_object_add(object, "urgent", json_object_new_boolean(urgent));
 	json_object_object_add(object, "sticky", json_object_new_boolean(c->is_sticky));
+	json_object_object_add(object, "fullscreen_mode", json_object_new_boolean(c->is_fullscreen));
```

Comment:
In i3, this is an integer, not a boolean.

Reply:
Done, by using json_object_new_int
1 = fullscreen on
0 = fullscreen off